engines/experiment/plugin/manager/rpc.go

```diff
@@ -0,0 +1,45 @@
+package manager
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/gojek/turing/engines/experiment/manager"
+	"github.com/gojek/turing/engines/experiment/plugin/shared"
+)
+
+// rpcClient implements ConfigurableExperimentManager interface
+type rpcClient struct {
+	shared.RPCClient
+}
+
+func (c *rpcClient) Configure(cfg json.RawMessage) error {
+	return c.Call("Plugin.Configure", cfg, new(interface{}))
+}
+
+func (c *rpcClient) GetEngineInfo() manager.Engine {
+	resp := manager.Engine{}
+	err := c.Call("Plugin.GetEngineInfo", new(interface{}), &resp)
+	if err != nil {
+		// err should be propagated upstream, but it's currently not
+		// possible as GetEngineInfo() on the manager.ExperimentManager
+		// interface doesn't return errors
+		fmt.Printf("plugin errors: %v", err)
+	}
+
+	return resp
+}
+
+// rpcServer serves the implementation of a ConfigurableExperimentManager
+type rpcServer struct {
+	Impl ConfigurableExperimentManager
+}
+
+func (s *rpcServer) Configure(cfg json.RawMessage, _ *interface{}) (err error) {
+	return s.Impl.Configure(cfg)
+}
+
+func (s *rpcServer) GetEngineInfo(_ interface{}, resp *manager.Engine) error {
```

Comment:
What about the rest of the experiment manager methods other than `GetEngineInfo`? Will they be implemented in a future PR ?

Reply:
Currently `ExperimentManager` interface has only a [single `GetEngineInfo` method.](https://github.com/gojek/turing/blob/c6884d5464439584f447927e23707732407282a8/engines/experiment/manager/manager.go#L15) and this is what is implemented here. 
As for the `StandardExperimentManager` and `CustomExperimentManager`, I'll need to give it another thought, as I'm not sure if the current way of casting the types through the `adapter` package is the best way with respect to the plugin implementation.

Reply:
Okay. Got it.